source/plugins/go/src/oms.go

```diff
@@ -1149,6 +1152,141 @@ func UpdateNumTelegrafMetricsSentTelemetry(numMetricsSent int, numSendErrors int
 	ContainerLogTelemetryMutex.Unlock()
 }
 
+func processIncludes(kubernetesMetadataMap map[string]interface{}, includesList []string) map[string]interface{} {
```

Comment:
Did you test with short lived (for example, 10s, 30s) Kubernetes jobs/cronjobs to see if filter plugin able to collect metadata?

Reply:
yes it works fine 
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/e474d173-8bdc-4ad2-80f1-4e068913637c)
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/9b21befe-b8fd-4925-ba76-2095c7aae50e)